docs/platform-parts/container/container-integration.md

```diff
@@ -216,18 +214,28 @@ To add a container using Carthage:
    carthage bootstrap --no-build --platform ios
    ```
 
-##### Manually adding a container
-
-To manually add a container:
+**Use git to clone container**
```

Comment:
When not used as a shell command, but used in a sentence like this, Git is capitalized. [[official docs](https://git-scm.com/)]